src/content/introduction.mdx

```diff
@@ -1,16 +1,20 @@
 ---
 title: Introduction
-description: Welcome to the initial draft of the Atom Learning design-system documentation
+description: Welcome to the Atom Learning design system
 ---
 
-It is divided up into three main parts:
+![badge](https://img.shields.io/npm/v/@atom-learning/components)
+![badge](https://img.shields.io/github/workflow/status/Atom-Learning/components/Test%20&%20validate)
+![badge](https://img.shields.io/bundlephobia/minzip/@atom-learning/components)
+
+The design system is divided up into three main parts:
 
 - A set of rules & guidelines for applying the system within the context of Atom Learning
 - Information on the theme layer and underlying visual system that powers the design-system
 - Technical documentation on our component library
 
 Our main focus for this initial release has been to create the in-depth technical documentation for our [component library](/components/introduction). There is also an initial draft of the [theme](/theme/introduction) content written, for use mainly as a visual reference for the scales and systems that form our design system.
 
-Finally, we're aiming to add a number of guidelines that talk not only about the components and how to use them, but how best to combine them or use them in context for a greater user experience. We also hope to include detailed guidance on more specific UX patterns like data capture through forms, consistent navigation patterns, tone of voice, how to best handle error & empty states, how to prioritise interface elements, and others.
+We're aiming to add a number of guidelines soon that talk not only about the components and how to use them, but also how best to combine them and use them in context for a greater user experience. We hope to include detailed guidance on more specific UX patterns like data capture through forms, consistent navigation patterns, tone of voice, how to best handle error & empty states, how to prioritise interface elements, and others.
 
 We welcome feedback so please provide comments or bug reports on our [Github repository](https://github.com/Atom-Learning/components/issues).
```

Comment:
Should this intro mention something about how the code is divided into 3 different repos? I feel we haven't explained it anywhere and it is a fundamental design decision people will need to get started

Reply:
Yeah I think that's a good shout, I'll update this now